### PR TITLE
docs: created doc for stepper motor experiment

### LINKED ISSUE #820: No documentation for stepper moter experiment
**Actual Behaviour**

The documentation for the stepper motor experiment is absent although the experiment setup is implemented.

**Expected Behaviour**

The documention should be there.

**Steps to reproduce it**

Make the experiment documents.

**LogCat for the issue**

Not applicable.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #820 

Changes: Added brief documentation for the experiment.

Screenshots for the change: 
screenshot_20180509-022644

APK for testing: [Compress the *.apk file into a rar or zip file and upload it here]
[app-debug.apk.tar.gz](https://github.com/fossasia/pslab-android/files/1985621/app-debug.apk.tar.gz)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal This is just the basic document for the experiment. Let me know what else to add in it.

File: app/src/main/assets/DOC_HTML/apps/J_stepper.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Stepper Motors
+---
+
+* Connect the stepper motor with the SQR pins appropriately.
```
Comment (CloudyPadmal) [inline app/src/main/assets/DOC_HTML/apps/J_stepper.md]: `Appropriately` means how? Can you please go through the implemented code and try to derive the documentation accordingly?

**Comment by CloudyPadmal** [comment]
Hello @abhinavraj23 ! Any update on this PR?

**Comment by abhinavraj23** [comment]
@CloudyPadmal in the implementation section it just takes the number of steps and direction of movement, I couldn't really derive anything about the pins used.

**Comment by CloudyPadmal** [comment]
@abhinavraj23 if you are a user and someone tells you `Connect the stepper motor with the SQR pins appropriately`, how would you connect the pins? It's very confusing. We have to give direct instructions. Mere documentation is not enough

**Comment by abhinavraj23** [comment]
@CloudyPadmal So currently as our planes are changed I am temporarily closing this PR.

**Pull request closed without merging**